packages/common-all/src/types/configs/commands/commands.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  genDefaultInsertNoteLinkConfig,
+  InsertNoteLinkConfig,
+  INSERT_NOTE_LINK,
+} from "./insertNoteLink";
+import { genDefaultLookupConfig, LookupConfig, LOOKUP } from "./lookup";
+
+/**
+ * Namespace for all command related configurations
+ */
+export type DendronCommandConfig = {
+  lookup: LookupConfig;
+  insertNoteLink: InsertNoteLinkConfig;
+};
+
+/**
+ * Constants holding all command config related {@link DendronConfigEntry}
+ */
+export const COMMANDS = {
```

Comment:
thoughts between using constants vs enums? You can use `Object.values` to get all enum values

Reply:
The values in the object are also constants that themselves holding other constants or functions that produce constants.

Are you suggesting we have nested enums (not sure if that's a thing)? or we flatten it out and have a giant enum that holds all possible labels and descriptions?